packages/experimental/Button/src/FabButton/FabButton.tsx

```diff
@@ -0,0 +1,24 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View } from 'react-native';
+import { fabButtonName, FabButtonType } from './FabButton.types';
+import { Text } from '@fluentui-react-native/experimental-text';
+import { compose, UseSlots } from '@fluentui-react-native/framework';
+import { Icon } from '@fluentui-react-native/icon';
+import { ButtonCoreProps } from '../Button.types';
+
+export const FabButton = compose<FabButtonType>({
+  displayName: fabButtonName,
+  slots: {
+    root: View,
+    icon: Icon,
+    content: Text,
+  },
+  render: (_userProps: ButtonCoreProps, _useSlots: UseSlots<FabButtonType>) => {
+    return (_final: ButtonCoreProps, ..._children: React.ReactNode[]) => {
+      return null;
```

Comment:
Maybe
``` jsx
return (
<Text>Not implemented for this platform</Text>
);
```

Reply:
I'd rather not since this will make it so that platforms that don't use the FAB but want to just have one copy of code across platforms suddenly get text where there shouldn't be any. I guess it's unlikely that this case would happen, but to me it seems odd for a component to suddenly generate text,

Reply:
I see. It's either they get nothing rendering or random text. I was thinking maybe we have some simple <NYI> aka <Not Yet Implemented> type component, but that's probably out of scope for this review and you make a fair point.